app/gui/view/documentation/assets/styles.css

```diff
@@ -0,0 +1,236 @@
+/* Custom CSS rules used by the documentation panel. */
+
+/* Common parts. */
+
+:root {
+  --type-name-color: #9640da;
+  --module-name-color: #a239e2;
+  --methods-header-color: #1f71d3;
+  --method-name-color: #1f71d3;
+  --types-header-color: #1f71d3;
+  --examples-header-color: #6da85e;
+  --important-background-color: #edefe7;
+  --info-background-color: #e6f1f8;
+  --example-background-color: #e6f1f8;
+  --docs-background-color: #fcfeff;
+  --docs-text-color: #434343;
+  --tag-background-color: #f5f5f5;
+  --caption-background-color: #0077f6;
+}
```

Comment:
As I've written in the previous comment, all variables / styles need to be prefixed with "enso-docs", otherwise they can override custom user-styles, e.g. in custom user visualizations. If I'm correct, these variables live in a global scope and can alias user defined ones, am I right?